apps/demos/Demos/Splitter/Overview/Vue/App.vue

```diff
@@ -38,6 +42,7 @@
               min-size="5%"
               text="Nested Right Pane"
               template="paneContent"
+      tabindex="0"
```

Comment:
It should be formatting issue. Repair formatting please

Reply:
[6d9726b](https://github.com/DevExpress/DevExtreme/pull/27306/commits/6d9726b5c1f6e803a0e4b8aac939c51b5a3e9d2a)